Gruntfile.js

```diff
@@ -18,14 +18,41 @@ module.exports = function( grunt ) {
   grunt.initConfig({
     pkg: grunt.file.readJSON( "package.json" ),
 
-    csslint: {
-      lax: {
+    watch: {
```

Comment:
If we do any front-end dev then we'll need to rebuild to test our changes won't we? This is a convenience tool for when we're patching the client-side code.